notebook/supermusr-plotting-frames.ipynb

```diff
@@ -0,0 +1,127 @@
+     },
+     "execution_count": 42,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Grab the frame's data\n",
+    "lower_limit = frame_list[frame]\n",
+    "upper_limit = frame_list[frame+1]\n",
+    "frame_data = detector_data[detector_to_plot][lower_limit:upper_limit]\n",
+    "frame_data"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%matplotlib qt\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "fig, ax = plt.subplots()\n",
```

Comment:
Could the plot be titled with the timestamp and frame number and have axis labels (for now x should just be "sample", I now realise the stream has enough information to allow conversion to time, which would be more useful, but a job for another day given I do not write it to the file. y is just a unitless "intensity", well, not really unitless, but for this purpose it effectively is unitless).

Reply:
I am fairly sure X is not sample, it is time.